python/source/org/libreoffice/ide/eclipse/python/test/JavaExportPageControlTest.java

```diff
@@ -0,0 +1,63 @@
+/*************************************************************************
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program.
+ * If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright: 2010 by Cédric Bosdonnat
+ *
+ * All Rights Reserved.
+ *
+ ************************************************************************/
+package org.libreoffice.ide.eclipse.python.test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+import org.libreoffice.ide.eclipse.python.export.JavaExportPageControl;
+
+/**
+ * Unit test for the Manifest export page part UI controller.
+ */
+public class JavaExportPageControlTest {
```

Comment:
I think you can remove all tests as they don't apply anymore.

Reply:
This particular file was removed when the Ant Export package was being removed.  It belongs to the package : org.libreoffice.ide.eclipse.python.Test

Removed all these packages since the Test written were for Java
org.libreoffice.ide.eclipse.python.tests
org.libreoffice.ide.eclipse.python.tests.base
org.libreoffice.ide.eclipse.python.tests.helper
org.libreoffice.ide.eclipse.python.tests.uno